##############
# Since we are installing all the needed packages, there is not need for a
# virtual environment
##############
diff --git a/run.sh b/run.sh
index 0906c41..4546671 100755
--- a/run.sh
+++ b/run.sh
@@ -2,8 +2,6 @@

 cd "$(dirname "$0")"

-. venv/bin/activate
-

 export FLASK_APP=episodes
 export FLASK_ENV=development
diff --git a/scripts/check.sh b/scripts/check.sh
index 7b21d81..8ae4141 100755
--- a/scripts/check.sh
+++ b/scripts/check.sh
@@ -6,8 +6,6 @@ cd "$(dirname "$0")"
 # Jump back to the root
 cd ..

-. venv/bin/activate
-
 cd episodes/

 while true
diff --git a/scripts/controller.sh b/scripts/controller.sh
index 6cf52a5..aaae636 100755
--- a/scripts/controller.sh
+++ b/scripts/controller.sh
@@ -6,8 +6,6 @@ cd "$(dirname "$0")"
 # Jump back to the root
 cd ..

-. venv/bin/activate
-
 rm instance/cast_socket

 python episodes/controller.py

